src/ensembl/src/shared/badged-button/BadgedButton.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import defaultStyles from './BadgedButton.scss';
+import classNames from 'classnames';
+
+type Props = {
+  children: React.ReactChild;
+  badge?: string | number | undefined;
```

Comment:
On a related note, if `badge` prop is not a component, but something that will be placed inside a badge, then perhaps the `badgeContent` name will be more suitable?

Reply:
I thought` badge` will be simpler but `badgeContent` seems to make it clear. I've updated it.